automatic/mkdocs-material/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,5 @@
+Update-SessionEnvironment
+
+$version = '0.0.0'
```

Comment:
Is this correct? Shouldn't it be 0.2.4, or will it get automatically updated with au afterwards?